plugins/by-name/lsp-format/default.nix

```diff
@@ -91,28 +90,40 @@ lib.nixvim.plugins.mkNeovimPlugin {
 
   extraConfig = cfg: {
     warnings = lib.nixvim.mkWarnings "plugins.lsp-format" {
-      when = !config.plugins.lsp.enable;
+      when = !config.plugins.lspconfig.enable;
       message = ''
-        This plugin requires `plugins.lsp` to be enabled.
+        This plugin requires `plugins.lspconfig` to be enabled.
```

Comment:
```suggestion
        This plugin requires either `plugins.lsp` or `plugins.lspconfig` to be enabled.
```

Reply:
Updated. On the use of the word "either": is nixvim enforcing uniqueness here? Or is OK to have both `plugins.lsp` and `plugins.lspconfig` enabled?

Reply:
> is nixvim enforcing uniqueness here?

Yes, that's done in the `plugins.lspconfig` module:
https://github.com/nix-community/nixvim/blob/4016854bccf192d6a5d7fdd5860571dfa1ece55c/plugins/by-name/lspconfig/default.nix#L50-L64